.travis.yml

```diff
@@ -18,6 +18,7 @@ install:
   - pip install flake8
 
 before_script:
+  - isort --check
```

Comment:
`isort --check` only outputs if the file is correct/incorrect. I believe `isort --check --diff` would be a better choice. This way we'll get more details on why the test failed

Reply:
We need to define a standard and stick to it. The `mypy-lang` doesn't belongs to test suite but it's in `requirements-test.txt`.

I'd like to suggest rename `requirements-test.txt` to `requirements-ci.txt` and put: `flake8`, `mypy-lang`, `isort`, etc inside it.

Reply:
I consider these `lint` command as `test` environment on my projects. On `ci` I run `make lint`...
It's a part on development and the developer should run this command locally... my opinion.